sql/database.sql

```diff
@@ -9582,3 +9587,10 @@ CREATE TABLE `product_registration` (
   PRIMARY KEY (`registration_id`)
 ) ENGINE=InnoDB;
 
+-- Table to copy log contents for audit log tamper resistance check.
+DROP TABLE IF EXISTS `log_validator`;
+CREATE TABLE `log_validator` (
+  `log_id` bigint(20) NOT NULL,
+  `log_checksum` longtext DEFAULT NULL,
```

Comment:
remove default NULL here(this is not legal in newer mysql versions):
``````
`log_checksum` longtext,